src/Features/VisualBasic/Portable/UseAutoProperty/VisualBasicUseAutoPropertyCodeFixProvider.vb

```diff
@@ -36,7 +37,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseAutoProperty
 
             Dim initializer = Await GetFieldInitializer(fieldSymbol, cancellationToken).ConfigureAwait(False)
             If initializer.equalsValue IsNot Nothing Then
-                statement = statement.WithInitializer(initializer.equalsValue)
+                statement = statement.WithTrailingTrivia(SyntaxFactory.Space) _
+                    .WithInitializer(initializer.equalsValue) _
+                    .WithTrailingTrivia(statement.GetTrailingTrivia.Where(Function(x) x.Kind <> SyntaxKind.EndOfLineTrivia)) _
+                    .WithAppendedTrailingTrivia(initializer.equalsValue.GetTrailingTrivia())
```

Comment:
If I don't add the Space it looks like `Public Property P() As Boolean= True`  Is there any issue with adding the space in this way?  In syntax visualizer it looks like the space should be trailing trivia on the keyword Boolean.

Also, is this an acceptable way to concatenate the comments from both the property and the field?

Reply:
1. the way you're doing it is acceptable. 
2. another way would be to do this:

`WithInitializer(initializer.equalsValue.WithPrependedLeadingTrivia(SyntaxFactory.ElasticTrivia))`

This will get the formatter to format the space between these two nodes however is best appropriate.

Reply:
Indeed :)